var/spack/repos/builtin/packages/neuron/package.py

```diff
@@ -161,8 +169,13 @@ def get_compiler_options(self, spec):
         if self.spec.satisfies('%pgi'):
             flags += ' ' + self.compiler.pic_flag
 
-        return ['CFLAGS=%s' % flags,
-                'CXXFLAGS=%s' % flags]
+        options = ['CFLAGS=%s' % flags,
+                   'CXXFLAGS=%s' % flags]
+
+        if spec.satisfies('+mpi'):
+            options.extend(['MPICC=%s' % spec['mpi'].mpicc,
+                            'MPICXX=%s' % spec['mpi'].mpicxx])
```

Comment:
this function name suggest to provide ` compiler options`. its fine to set compilers as well but could be move out of this.